### PR TITLE
Temporarily check for build and build-tools in GKE tests.

### PR DESCRIPTION
**Pull request by spxtr**
https://github.com/kubernetes/kubernetes/pull/35453

@mikedanese

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/906)

### CODE REVIEW

**Comment by mikedanese** [comment]
LGTM

File: jobs/pull-kubernetes-e2e-gke-gci.sh [hunk 1]
```diff
@@ -34,7 +34,16 @@ export JENKINS_USE_LOCAL_BINARIES=y
 export KUBE_FASTBUILD=true
 ./hack/jenkins/build.sh
 
-version=$(source build/util.sh && echo $(kube::release::semantic_version))
+# TODO(spxtr): once https://github.com/kubernetes/kubernetes/pull/35453 is in,
```
Comment (ixdy) [inline jobs/pull-kubernetes-e2e-gke-gci.sh]: We can't do that until we're no longer testing anything older than 1.5 (so a year or so).

Comment (spxtr) [inline jobs/pull-kubernetes-e2e-gke-gci.sh]: Yeah, that's gross.

**Comment by fejta** [comment]
Feel free to merge this w/o me. I won't have github access for the next
O(hour)

**Pull request merged**